### ISSUE TITLE
setIntervalStart is overwriting the intervalEndDate

### ISSUE THREAD

**Issue by dantonyblanco** [kylestetz/CLNDR#256]
this.intervalEnd = this.intervalStart.clone()
                .add(timeOpt- 1, 'days')
                .endOf('day');

This line on the setIntervalStart method its trying the get the days from the object instead of accessing the days property.
It should be:
.add(timeOpt.days - 1)

**Comment by mikegioia** [comment]
@dantonyblanco Could you send a more detailed update, possibly with a way to reproduce the problem, the expected output, and the actual output?

**Comment by dantonyblanco** [comment]
Sure, sorry I did not include more details before. I was working on something. Anyhow.
When using clndr we are trying to change the date range we are displaying. 
We have datepicker and the date that you select will be the first of the 5 days the our calendar shows.
When we try to set the new intervalStart, clndr will correctly go to the selected day. But instead of showing 5 days it shows only the day we selected in the datepicker.

I debugged this and found that the issue is in the code i commented above once the correct property was used everything worked nicely.
